02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<p>might want to put a screenshot of the actual result so those that can't run this can still see the result. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='86'/>